plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPreferencesModel.java

```diff
@@ -99,6 +102,14 @@ public void setStopPreviousVersion(boolean stopPreviousVersion) {
     this.stopPreviousVersion = stopPreviousVersion;
   }
 
+  public boolean isConfigDeploy() {
```

Comment:
`shouldDeployConfigurationFiles()`?  (and should you choose to accept this mission, perhaps rename `isAutoPromote()` and `isStopPreviousVersion()`?)